Tiltfile

```diff
@@ -7,11 +7,13 @@ DOCKER_REGISTRY="localhost:63628/"
 v1alpha1.extension_repo(name='grafana-tilt-extensions', url='https://github.com/grafana/tilt-extensions')
 v1alpha1.extension(name='grafana', repo_name='grafana-tilt-extensions', repo_path='grafana')
 load('ext://grafana', 'grafana')
+load('ext://configmap', 'configmap_create')
 
 allow_k8s_contexts(["kind-kind"])
 
 docker_build("localhost:63628/oncall/engine:dev", "./engine", target = 'prod')
-docker_build("localhost:63628/oncall/ui:dev", "./grafana-plugin", dockerfile = './grafana-plugin/Dockerfile.dev')
+# Build the plugin in the background
+docker_compose('grafana-plugin/docker-compose-dev.yaml')
```

Comment:
you can also pass in [`ui.enabled` to the helm chart](https://github.com/grafana/oncall/blob/dev/helm/oncall/values.yaml#L512-L515) and this will spin up a UI container for hot-reloading purposes